src/software/ai/hl/stp/play/penalty_kick_play_test.cpp

```diff
@@ -90,6 +81,15 @@ TEST_F(PenaltyKickPlayTest, test_penalty_kick_take)
         [shooter_id](std::shared_ptr<World> world_ptr,
                      ValidationCoroutine::push_type& yield) {
             robotsAvoidBall(1, {shooter_id}, world_ptr, yield);
+        },
+        [shooter_id](std::shared_ptr<World> world_ptr,
+                     ValidationCoroutine::push_type& yield) {
+            // Wait 2 seconds for robots to start moving adequately far away from the ball
+            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(2))
+            {
+                yield();
+            }
+            robotsAvoidBall(1, {shooter_id}, world_ptr, yield);
```

Comment:
I think these should be moved to line 47?

Reply:
good catch